app/controllers/users_controller.rb

```diff
@@ -17,7 +20,8 @@ def show
     @item = Item.new
   end
 
-  def new
+  def index
```

Comment:
index is normally used to show a LIST of elements, in this case users. If you are only going to show one, that's a show action. Index doesn't have params, so how come you have a user_id?